gallery/src/styles/NxFormLayout/NxFormHorizontalLayoutExample.tsx

```diff
@@ -8,49 +8,46 @@ import React from 'react';
 
 import { NxCheckbox } from '@sonatype/react-shared-components';
 import { NxRadio } from '@sonatype/react-shared-components';
-
-import { NxFontAwesomeIcon } from '@sonatype/react-shared-components';
-import { faCalendar } from '@fortawesome/free-solid-svg-icons';
+import { NxButton } from '@sonatype/react-shared-components';
+import { NxStatefulTextInput } from '@sonatype/react-shared-components';
 
 export default function NxFormLayoutExample() {
+  function validator(val: string) {
+    return val.length ? null : 'Must be non-empty';
+  }
+
   return (
     <form className="nx-form">
       <div className="nx-form-row">
```

Comment:
It seems a little unfortunate to me that we need a separate element for a strictly layout concern.  Would it not work to instead create `nx-form-group--inline` (or `nx-label--inline` if `nx-form-group` is removed)?

Reply:
> It seems a little unfortunate to me that we need a separate element for a strictly layout concern. Would it not work to instead create `nx-form-group--inline` (or `nx-label--inline` if `nx-form-group` is removed)?

By enclosing the groups inside a row we control how many and which appear side-by-side. There are ways that we could do that and still drop the row, but that would add complexity elsewhere. I also prefer this because it mimics table layout and therefore should be familiar to most.